openrtb-core/src/main/java/com/google/openrtb/json/OpenRtbJsonReader.java

```
@@ -740,7 +740,7 @@ protected void readContentField(JsonParser par, Content.Builder content, String
         content.setVideoquality(VideoQuality.valueOf(par.getIntValue()));
         break;
       case "context":
-        content.setContext(Context.valueOf(par.getIntValue()));
+        content.setContext(Context.valueOf(par.getValueAsInt()));
```

Thanks for spotting this! I'll only ask for some improvements. :)

1) Put a comment here explaining that we need this different code due to backwards compatibility with 2.2. (At some point in the future, maybe by 2.4 or whenever we don't have to care about 2.2 anymore, it would make sense to clean it up and revert to just `getIntValue()`.)

2) This doesn't always work well because `getValueInt()` will silently return 0 if the field doesn't contain a parseable number. If this happens, `Context.valueOf(...)` will return null because there's no enumerated value for 0. and then, `content.setContext(...)` will throw `NullPointerException`. So we need to handle this better. I suggest just keep using `getIntValue()`, but put all code inside a try/catch for `JsonParseException` and just ignore this exception.